config/templates/Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG BASE_IMAGE
 FROM $BASE_IMAGE as armbian_builder
-ARG GCC_COMPILER
-ENV GCC_COMPILER $GCC_COMPILER
+ARG ARM_GCC_COMPILER
```

Comment:
Propose to add default values:
ARG ARM_GCC_COMPILER='g++-11-arm-linux-gnueabihf'

Reply:
then dockerfile still easy can be used separate, with manual build